plugins/available/battery.plugin.bash

```diff
@@ -2,15 +2,23 @@ cite about-plugin
 about-plugin 'display info about your battery charge level'
 
 ac_adapter_connected(){
-    if command_exists acpi;
-    then
-        acpi -a | grep "on-line"
-        if [[ "$?" -eq 0 ]]; then
-           return 1
-        else
-           return 0
-        fi
```

Comment:
Yes, that's true. So after this change, both OS X and Linux are returning this:

* 0: AC adapter connected
* 1: AC adapter not connected